### PR TITLE
LPAL-232 Title lost dropdown

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

https://opgtransform.atlassian.net/browse/LPAL-232

Fixes LPAL-232

## Approach

Debugged how options were added to the title select element; noticed that when the JS console showed an analytics error, the options didn't render; put a guard into the JS function so that it wouldn't call a non-existent method, which allowed the JS to run properly.

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [x] The product team have tested these changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -199,3 +199,10 @@ endif
 .PHONY: restitch
 restitch:
 	cypress/stitch.sh
+
+# Start S3 Monitor and call "cypress open";
+# this requires a globally-installed cypress
+.PHONY: cypress-open
+cypress-open:
```
Comment (williamfalconeruk) [inline Makefile:206]: noice 👍🏻

**Review by williamfalconeruk — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -199,3 +199,10 @@ endif
...
+	aws-vault exec moj-lpa-dev -- python3 cypress/S3Monitor.py &
+	CYPRESS_userNumber=`node cypress/userNumber.js` cypress open
```
Comment (nickdavis2001) [inline Makefile:208]: Nice :)

File: cypress/userNumber.js [hunk 3]
```diff
@@ -1,11 +1,7 @@
...
-var date = new Date();
-var userNumber = date.getTime() + "" + random;
-console.log(userNumber);
+// This makes a user number for tests.
+// It needs to happen before startup rather than in support/index.js so
+// that we have the same user number for all feature files under CI.
+// This code is therefore called from start.sh.
+// For safety, for now we generate userNumber using
+// the exact same js code that the Casper tests did.
+console.log("" + (new Date()).getTime() + Math.floor(Math.random() * 999999999));
```
Comment (nickdavis2001) [inline cypress/userNumber.js:7]: Do ya really think that's more readable than what was there before?? ;)

Comment (townxelliot) [reply cypress/userNumber.js:7]: No, I think I'll revert it

**Review by nickdavis2001 — APPROVED** [review]
lgtm

**Pull request merged**